cmd/coordinator/run.go

```diff
@@ -53,24 +55,37 @@ func run(validator quote.Validator, issuer quote.Issuer, sealDir string, sealer
 	meshServerAddr := util.Getenv(config.MeshAddr, config.MeshAddrDefault)
 	promServerAddr := os.Getenv(config.PromAddr)
 
+	// Create Prometheus resources and start the Prometheus server.
+	var promRegistry *prometheus.Registry
+	var promFactory promauto.Factory
+	if promServerAddr != "" {
```

Comment:
We don't consider the Prometheus logging security-relevant because all information logged can essentially be obtained by observing the cluster as an untrusted party.
So for the sake of simplicity, I'd keep it out of the manifest.